eks/cuda-vector-add/cuda-vector-add.go

```diff
@@ -0,0 +1,231 @@
+package cuda_vector_add
+
+import (
+	"fmt"
+	eks_tester "github.com/aws/aws-k8s-tester/eks/tester"
```

Comment:
let's move all non-standard packages below, and run `gofmt` (`./scripts/fmt.sh`)

Reply:
Fixed.